### PR TITLE
BUG: avoid certificate experation

### PR DESCRIPTION
**Pull request by aburrell**
Avoid certificate expiration by using the `verify` flag set to False.

Addresses an issue identified at APL, where their certificate had expired.

# Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Unit tests

## Test Configuration
* Operating system: OS X Big Sur
* Version number: Python 3.10
* Any details about your local setup that are relevant: develop branch of pysat

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

**Comment by aburrell** [comment]
- [ ] fix broken links
- [ ] update intersphinx mapping
- [ ] fix whatever is up with the GA tests

**Comment by aburrell** [comment]
@jklenzing any idea how I can get the coverage analysis to run?

File: pysatSpaceWeather/instruments/methods/f107.py [hunk 1]
```diff
@@ -373,14 +373,17 @@ def calc_f107a(f107_inst, f107_name='f107', f107a_name='f107a', min_pnts=41):
...
     # Resample to the original frequency, if it is not equal to 1 day
     freq = pysat.utils.time.calc_freq(f107_inst.index)
-    if freq != "86400S":
+    if freq != "86400s":
         # Resample to the desired frequency
-        f107_fill = f107_fill.resample(freq).ffill()
+        if pds.to_timedelta(freq) < pds.to_timedelta("86400s"):
+            f107_fill = f107_fill.resample(freq).ffill()
+        else:
+            f107_fill = f107_fill.resample(freq).asfreq()
```
Comment (aburrell) [inline pysatSpaceWeather/instruments/methods/f107.py:381]:
```suggestion
        f107_fill = f107_fill.resample(freq).ffill()

**Comment by jklenzing** [comment]
> @jklenzing any idea how I can get the coverage analysis to run?

Here's what is working for pysatSeasons: https://github.com/pysat/pysatSeasons/pull/43

**Review by jklenzing — CHANGES_REQUESTED** [review]
Make sure the updates to coveralls are included in the pysat_rc.yml

**Comment by jklenzing** [comment]
```
FAILED pysatSpaceWeather/tests/test_instruments.py::TestInstruments::test_load_w_pad[pad0-inst_dict16] - AssertionError: Padding should have left data and didn't
FAILED pysatSpaceWeather/tests/test_instruments.py::TestInstruments::test_load_w_pad[pad0-inst_dict25] - AssertionError: Padding should have left data and didn't
FAILED pysatSpaceWeather/tests/test_instruments.py::TestInstruments::test_load_w_pad[pad1-inst_dict16] - AssertionError: Padding should have left data and didn't
FAILED pysatSpaceWeather/tests/test_instruments.py::TestInstruments::test_load_w_pad[pad1-inst_dict25] - AssertionError: Padding should have left data and didn't
```
Padding failures throughout

**Comment by aburrell** [comment]
@jklenzing that was an unlucky timing issue with one of the realtime files.  Reran tests and now have only expected failures.

**Review by jklenzing — APPROVED** [review]

**Pull request merged**